starter/js/atm.js

```diff
@@ -1,43 +1,54 @@
 //Begin with the document ready function
+$(document).ready(function(){
 
+}
     //Checking account deposit function
-
       //On click of the depositChecking button
-
+$("#depositChecking").click(function(event){
         //Get value from the amountChecking input field
-
+   var amountBalance = $("#amountChecking").val();
         //Take that value and add it to the existing value in the checkingBalance div
-
+   var checkingBalance = amountBalance.val();
     //Checking account withdrawl funtion
-
+});
+event();
       //On click of the withdrawChecking button
-
+$("#withdrawChecking").click(function(event){
         //Get value from the amountChecking input field
-
+var amountBalance = $("#amountChecking").val();
         // If that value is greater than the value in the account ignore that action
         // In other words if this would put the account into a negative balance do not allow it
-
+if (amountBalance < 0) {
+  return prompt("Not allowed!");
+} else {
+  checkingBalance - amountBalance;
+};
         //Else subtract that value from the current amount in the checking account
```

Comment:
You are missing the close to this click event. Make sure to indent so you can see where things open and close. You would need `});` on this line.